app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(input_str: str) -> None:
+    old_name, new_name = input_str.strip("cp ").split()
```

Comment:
The code does not check if the input string contains exactly three elements separated by spaces and starts with 'cp'. According to the checklist, you should ensure the input string meets this format requirement.